File: packages/tasks-gen/snippets-fixtures/conversational-llm-non-stream/python/huggingface_hub/0.together.py
from huggingface_hub import InferenceClient, get_token

client = InferenceClient(
    api_key=get_token(),
)

completion = client.chat.completions.create(
    model="meta-llama/Llama-3.1-8B-Instruct:together",
    messages=[
        {
            "role": "user",
            "content": "What is the capital of France?"
        }
    ],
)

print(completion.choices[0].message)

File: packages/tasks-gen/snippets-fixtures/conversational-llm-non-stream/js/openai/0.together.js
import { OpenAI } from "openai";

const client = new OpenAI({
	baseURL: "https://router.huggingface.co/together/v1",
	apiKey: process.env.HF_TOKEN,
});

const chatCompletion = await client.chat.completions.create({
	model: "<together alias for meta-llama/Llama-3.1-8B-Instruct>",
    messages: [
        {
            role: "user",
            content: "What is the capital of France?",
        },
    ],
});

console.log(chatCompletion.choices[0].message);